README.md

```diff
@@ -286,6 +186,6 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
kindly fix the broken link in this section to make your readme more professional